apps/fluent-tester/src/TestComponents/CheckboxExperimental/CheckboxTest.tsx

```diff
@@ -147,15 +160,23 @@ const TokenCheckbox: React.FunctionComponent = () => {
   );
 };
 
+const UnSupportedOnAndroid = () => {
```

Comment:
We should finalize on a uniform way of handling the cases of unsupported props, button current is just removing off those segments.
If we decide to go with the approach used here, we should move it to someplace common between all components

Reply:
yes currently we're going with Platform.select() and not rendering the unsupported sections at all.